cmd/kubeadm/app/constants/constants.go

```diff
@@ -164,10 +164,10 @@ const (
 	KubeletBaseConfigurationFile = "kubelet"
 
 	// MinExternalEtcdVersion indicates minimum external etcd version which kubeadm supports
-	MinExternalEtcdVersion = "3.0.14"
+	MinExternalEtcdVersion = "3.1.10"
```

Comment:
@luxas MinExternalEtcdVersion should be the supported etcd version of 1.9, right?

Reply:
cool